app/src/main/java/co/techmagic/hr/presentation/ui/activity/RequestTimeOffActivity.kt

```diff
@@ -356,7 +356,7 @@ class RequestTimeOffActivity : BaseActivity<RequestTimeOffViewImpl, RequestTimeO
             }
         }
 
-        override fun onCreateViewHolder(parent: ViewGroup?, viewType: Int): ViewHolder {
+        override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
             val view: View = LayoutInflater.from(parent!!.context).inflate(R.layout.item_requested_time_off, parent, false)
```

Comment:
```suggestion
            val view: View = LayoutInflater.from(parent.context).inflate(R.layout.item_requested_time_off, parent, false)
```

Reply:
I've tried to reduce the amount of fingerprint by applying these changes.